src/storage/store/engine.rs

```diff
@@ -826,11 +927,23 @@ impl ShardEngine {
                 .user_data_store
                 .merge(msg, txn_batch)
                 .map_err(|e| MessageValidationError::StoreError(e)),
-            MessageType::VerificationAddEthAddress | MessageType::VerificationRemove => self
+            MessageType::VerificationAddEthAddress => self
```

Comment:
@aditiharini one small note - There is no mention here of `VerificationAddEthSolana` when there should be. Am I wrong or is there a bug?